.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -157,14 +167,15 @@ body:
     attributes:
       label: Device model
       description: What device you are experiencing this problem on? Specify full device name along with the version of the operating system it's running.
-      placeholder: ex. Samsung Galaxy A22 (Android 12)
+      placeholder: Pixel 6 Pro API 36.0
 
   - type: dropdown
     id: acknowledgements
     attributes:
       label: Acknowledgements
       description: I searched for similar issues in the repository.
       options:
-        - 'Yes'
+        - Yes
+        - No
```

Comment:
I'm adding another option here so that users need to _actively_ select "Yes". If there's only option, it is automatically selected:

<img width="304" height="118" alt="Screenshot 2025-07-25 at 12 00 25" src="https://github.com/user-attachments/assets/2878b18a-c7ca-42f6-8838-7612cc213067" />